packages/web-twig/src/Resources/components/TextField/README.md

```diff
@@ -39,22 +40,23 @@ Without lexer:
-| `id`              | `string`                                                      | —       | yes      | Input and label identification                             |
-| `isDisabled`      | `bool`                                                        | `false` | no       | If true, input is disabled                                 |
-| `isFluid`         | `bool`                                                        | `false` | no       | If true, the element spans to the full width of its parent |
-| `isLabelHidden`   | `bool`                                                        | `false` | no       | If true, label is hidden                                   |
-| `isRequired`      | `bool`                                                        | `false` | no       | If true, input is required                                 |
-| `label`           | `string`                                                      | —       | yes      | Label text                                                 |
-| `message`         | `string`                                                      | `null`  | no       | Validation or help message                                 |
-| `name`            | `string`                                                      | `null`  | no       | Input name                                                 |
-| `placeholder`     | `string`                                                      | `null`  | no       | Input placeholder                                          |
-| `inputWidth`      | `number`                                                      | `null`  | no       | Input width                                                |
-| `type`            | `email`, `number`, `password`, `search`, `tel`, `text`, `url` | `text`  | no       | Input type                                                 |
-| `validationState` | `success`, `warning`, `error`                                 | `null`  | no       | Type of validation state                                   |
-| `value`           | `string`                                                      | `null`  | no       | Input value                                                |
+| Prop name           | Type                                                          | Default | Required | Description                                                |
+| ------------------- | ------------------------------------------------------------- | ------- | -------- | ---------------------------------------------------------- |
+| `class`             | `string`                                                      | `null`  | no       | Custom CSS class                                           |
+| `hasPasswordToggle` | `bool`                                                        | `false` | no       | If true, input has a password toggle                       |
```

Comment:
Please don't forget to mention it forces the `password` type.

Reply:
> TextAreaField now has the same prop.

No, it doesn't. It makes no sense to have this prop there, so it is the same as inputWidth (just TextField) or rows (just TextArea). But I move the prop to the correct place. 
![image](https://user-images.githubusercontent.com/7041807/212551808-63f7fe4b-581a-4ba8-bf72-018fe7b28a27.png)